Assets/Scripts/ScoreManager.cs

```diff
@@ -56,4 +59,8 @@ public static void AddScore(ScoreData score)
     }
 
     public static List<ScoreData> GetScoreData() { return Instance.racerScores; }
+    public int GetMinScore() { return MinTimeScore; }
```

Comment:
These methods should be static given that the ScoreManager is a singleton. You should be returning Instance.minTimeScore instead.

Also, to follow the coding style we established the variable names should begin with a non-capitalized letter.